sos/report/plugins/pacemaker.py

```diff
@@ -42,13 +43,21 @@ def setup_crm_shell(self):
         ])
 
     def setup_pcs(self):
+        pcs_pkg = self.policy.package_manager.pkg_by_name('pcs')
+
         self.add_copy_spec("/var/log/pcsd/pcsd.log")
         self.add_cmd_output([
             "pcs stonith sbd status --full",
             "pcs stonith sbd watchdog list",
             "pcs stonith history show",
-            "pcs property list --all"
         ])
+
+        pcs_version = '.'.join(pcs_pkg['version'])
```

Comment:
Try running `sos report -o pcaemaker` on a system *without* `pcs` package installed - you get

    TypeError: 'NoneType' object is not subscriptable

here (CI tests fail on this)

Reply:
Thanks again. How about this? I had added a return if the `pcs` package does not exist. All commands within the `setup_pcs()` depend on the `pcs` package, so I believe this should be fine. 
Of course, if there are any problems with this, I will make corrections as per your suggestion.

Reply:
Oh, sorry.
```
flake8 sos
sos/report/plugins/pacemaker.py:48:12: E111 indentation is not a multiple of 4
```
I will fix this and push it again.